### ISSUE TITLE
[Style] Change the theme buttons position on small devices

### ISSUE THREAD

**Issue by Rakesh9100** [Rakesh9100/Eventica#74]
### The both theme-changing buttons currently appears inside the hamburger menu at last on small devices. It should be placed beside the hamburger icon for better accessibility and visibility on small devices.

![Image](https://github.com/user-attachments/assets/472d92b6-339e-4b82-a484-f99c6b5aff6e)

### Checklist

- [x] I have checked the existing issues.
- [x] I follow [Contributing Guidelines](https://github.com/Rakesh9100/Eventica/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [x] I am a IWOC'25 contributor.

### Code of Conduct

- [x] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/Eventica/blob/main/.github/CODE_OF_CONDUCT.md)".

**Comment by swapnilyadv** [comment]
![Image](https://github.com/user-attachments/assets/11cd9115-d376-47d8-944f-68606ddc0574)
understand your issue working on it

**Comment by deepeshahlawat** [comment]
hey @Rakesh9100 I want to work on this issue. Can you please assign me?

**Comment by Rakesh9100** [comment]
Go ahead @swapnilyadv

**Comment by Rakesh9100** [comment]
PR update @swapnilyadv

**Comment by swapnilyadv** [comment]
I am busy in my clg practical so I need more time to complete this issue

**Comment by Rakesh9100** [comment]
More time means how many days @swapnilyadv

**Comment by swapnilyadv** [comment]
This Sunday I will fix this issue and check my pull request about page and
what I created readme file issue check once pls

**Comment by Sakshamjain98** [comment]
Please assign me under iwoc

**Comment by Rakesh9100** [comment]
You are already assigned to some issues in my other projects, complete those first and don't simply say to assign in every issue @Sakshamjain98

**Comment by Pratyush2529** [comment]
Hi @Rakesh9100 , I’d love to work on this issue under IWOC 3.0
Could you please assign it to me?

**Comment by Pratyush2529** [comment]
Hi @Rakesh9100 , I have tried to fix your issue. Is this what you were looking for? 

![Image](https://github.com/user-attachments/assets/e4dfe600-e9a1-4072-9c98-13dacec7dce8)

**Comment by Rakesh9100** [comment]
Yes, right, go ahead @Pratyush2529
